src/settings/settings.go

```diff
@@ -31,6 +31,7 @@ type Settings struct {
 	RedisPerSecondAuth         string `envconfig:"REDIS_PERSECOND_AUTH" default:""`
 	RedisPerSecondTls          bool   `envconfig:"REDIS_PERSECOND_TLS" default:"false"`
 	ExpirationJitterMaxSeconds int64  `envconfig:"EXPIRATION_JITTER_MAX_SECONDS" default:"300"`
+	LocalCacheSize             int    `envconfig:"LOCAL_CACHE_SIZE" default:"0"`
```

Comment:
What unit is this? Entries? Bytes? More comments and/or make the name more clear?

Reply:
Just changed the name, to make it clear.